### PR TITLE
add recipe card for each recipe and revise styling

### PR DESCRIPTION
**Pull request by matthewcotton**

### CODE REVIEW

File: src/pages/RecipePages/RecipeFeed.js [hunk 1]
```diff
@@ -1,11 +1,62 @@
...
+        <Row className="d-flex justify-content-center">
+          <Col className="text-left">
+            <h1>Recipes</h1>
+          </Col>
+          <Col className="text-right">
+            <RecipeButton btnText="Add Recipe" btnLink="/user/create-recipe" />
+          </Col>
+        </Row>
+        <Row className="justify-content-center">
+          <Col lg={10}>
```
Comment (TTBoiMike) [inline src/pages/RecipePages/RecipeFeed.js:51]: set lg={12} to make the recipes card full width of the Container element

File: src/pages/RecipePages/RecipeFeed.js [hunk 2]
```diff
@@ -1,11 +1,62 @@
...
+            <Container>
```
Comment (TTBoiMike) [inline src/pages/RecipePages/RecipeFeed.js]: remove <Container> as its applied on line 41

File: src/components/Recipes/RecipeCard.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Card } from "react-bootstrap";
+import { RecipeButton } from "./RecipeButton";
+import DefaultImg from "../../assets/images/edgar-castrejon-1SPu0KT-Ejg-unsplash.jpg";
+
+export const RecipeCard = ({ recipe }) => {
+  return (
+    <Card className="recipe-card">
```
Comment (TTBoiMike) [inline src/components/Recipes/RecipeCard.js:9]: set mx-0 as a class to remove left and right margins

File: src/components/Recipes/RecipeButton.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+export const RecipeButton = ({ btnText, btnLink }) => {
+  return (
+    <>
+      <Button className="button-main">
+        <Link to={btnLink}>{btnText}</Link>
```
Comment (TTBoiMike) [inline src/components/Recipes/RecipeButton.js:9]: if you move the link outside of the button element it won't apply the default styling

File: src/pages/RecipePages/RecipeFeed.js [hunk 5]
```diff
@@ -1,11 +1,62 @@
...
+        </Col>
+      </Row>
+    );
+  };
+
+  return (
+    <>
+      <Container className="pt-5 feed-container">
+        <Row className="d-flex justify-content-center">
+          <Col className="text-left">
```
Comment (TTBoiMike) [inline src/pages/RecipePages/RecipeFeed.js]: set xs={6} lg={6} - to set the left/right column in line with each other

**Review by TTBoiMike — COMMENTED** [review]
just some css styling points but nothing major

File: src/components/Recipes/RecipeButton.js (hunk 4 above)
Comment (matthewcotton) [reply src/components/Recipes/RecipeButton.js:9]: I have changed this button in the next branch so I won't change here.

**Comment by matthewcotton** [comment]
I have made all changes in the next branch as I want to minimise merge conflicts. I will raise a pull request for this branch soon.

**Pull request merged**